### ISSUE TITLE
Remote-SSH: Failed to connect to the remote extension host server (Error: No remote extension installed to resolve ssh-remote.)

### ISSUE THREAD

**Issue by GAZ082** [microsoft/vscode-remote-release#771]
- VSCode Version: 1.35.1
- Local OS Version: Windows 10
- Remote OS Version: Archlinux
- Remote Extension/Connection Type: SSH

Steps to Reproduce:

1. SSH-Remote connect to host.
2. Enter username@hostip.
3. Focus on the automatically opened terminal, enter password.
4. New VSCODE window opens. After a few seconds of trying to connect two errors show up:
```` 
Failed to connect to the remote extension host server (Error: No remote extension installed to resolve ssh-remote.)
```` 

```` 
Resolving the authority `ssh-remote` failed
```` 
Does this issue occur when you try this locally and all extensions are disabled?: Yes

Edit: The logs show nothing. Perhaps I'm not checking well, but the looked all the files in the log directory and nothing.

**Comment by roblourens** [comment]
That's strange. Does this happen every time? In the window with the error, do you see the Remote-SSH extension in the extension viewlet?

**Comment by OliverSchmitt** [comment]
Ihave exactly the same problem. In my case:

- VS Code Version: 1.35.0
- Local OS: Manjaro
- Remote OS: Ubuntu 18.04

it does happen every time and the Remote Development extension including the Remote-SSH extension are shown as installed.

**Comment by roblourens** [comment]
Can you try updating vscode (we just released 1.36) and try again?

**Comment by OliverSchmitt** [comment]
It works with the newly released 1.36 version. Thanks for your help.
